src/main.rs

```diff
@@ -1,7 +1,7 @@
 #![no_std]
 #![no_main]
 
-use core::mem::MaybeUninit;
+// use core::mem::MaybeUninit;
```

Comment:
We can just remove the line of code :)

Reply:
Great, thanks!

Reply:
I believe the comment is still there instead of removing the line :P

Reply:
Ups, sorry, I thought by resolving, it takes the suggestion. I make new commit later.